.github/actions/build-custom-image/action.yaml

```diff
@@ -31,16 +31,17 @@ runs:
       REPO: ${{ inputs.docker_repo }}
     shell: bash
     run: |
-      MANIFEST_NAME="./$(uuidgen).json"
+      METADATA_FILE="$(mktemp).json"
```

Comment:
I guess we need `--suffix` to make a `.json` file, or there will become two files: The file created by `mktemp` and a file beside it named `(mktemp).json`.

Reply:
I did not do that because `mktemp` on macOS does not have `--suffix`. That's a GNU specific option. I guess this is harmless. The name returned by `mktemp` is unique enough.